dDatabase/CDClientDatabase/CDClientTables/CDPhysicsComponentTable.cpp

```diff
@@ -4,32 +4,31 @@ void CDPhysicsComponentTable::LoadValuesFromDatabase() {
 	auto tableData = CDClientDatabase::ExecuteQuery("SELECT * FROM PhysicsComponent");
 	auto& entries = GetEntriesMutable();
 	while (!tableData.eof()) {
-		CDPhysicsComponent entry;
-		entry.id = tableData.getIntField("id", -1);
-		entry.bStatic = tableData.getIntField("static", -1) != 0;
-		entry.physicsAsset = tableData.getStringField("physics_asset", "");
-		UNUSED(entry->jump = tableData.getIntField("jump", -1) != 0);
-		UNUSED(entry->doublejump = tableData.getIntField("doublejump", -1) != 0);
-		entry.speed = tableData.getFloatField("speed", -1);
-		UNUSED(entry->rotSpeed = tableData.getFloatField("rotSpeed", -1));
-		entry.playerHeight = tableData.getFloatField("playerHeight");
-		entry.playerRadius = tableData.getFloatField("playerRadius");
-		entry.pcShapeType = tableData.getIntField("pcShapeType");
-		entry.collisionGroup = tableData.getIntField("collisionGroup");
-		UNUSED(entry->airSpeed = tableData.getFloatField("airSpeed"));
-		UNUSED(entry->boundaryAsset = tableData.getStringField("boundaryAsset"));
-		UNUSED(entry->jumpAirSpeed = tableData.getFloatField("jumpAirSpeed"));
-		UNUSED(entry->friction = tableData.getFloatField("friction"));
-		UNUSED(entry->gravityVolumeAsset = tableData.getStringField("gravityVolumeAsset"));
+		const uint32_t componentID = tableData.getIntField("id", -1);
+
+		auto& entry = entries[componentID];
+		entry.id = componentID,
+		entry.bStatic = tableData.getIntField("static", -1) != 0,
+		entry.physicsAsset = tableData.getStringField("physics_asset", ""),
+		UNUSED_ENTRY(entry.jump = tableData.getIntField("jump", -1) != 0,)
+		UNUSED_ENTRY(entry.doubleJump = tableData.getIntField("doublejump", -1) != 0,)
+		entry.speed = static_cast<float>(tableData.getFloatField("speed", -1)),
+		UNUSED_ENTRY(entry.rotSpeed = tableData.getFloatField("rotSpeed", -1),)
+		entry.playerHeight = static_cast<float>(tableData.getFloatField("playerHeight")),
+		entry.playerRadius = static_cast<float>(tableData.getFloatField("playerRadius")),
+		entry.pcShapeType = tableData.getIntField("pcShapeType"),
+		entry.collisionGroup = tableData.getIntField("collisionGroup"),
+		UNUSED_ENTRY(entry.airSpeed = tableData.getFloatField("airSpeed"),)
+		UNUSED_ENTRY(entry.boundaryAsset = tableData.getStringField("boundaryAsset"),)
+		UNUSED_ENTRY(entry.jumpAirSpeed = tableData.getFloatField("jumpAirSpeed"),)
+		UNUSED_ENTRY(entry.friction = tableData.getFloatField("friction"),)
+		UNUSED_ENTRY(entry.gravityVolumeAsset = tableData.getStringField("gravityVolumeAsset"),)
```

Comment:
why was every semi colon replaced with a comma?

Reply:
Genuinely unsure what happened there

Reply:
I actually remember now: These used to be in an aggregate constructor. Guess I forgot to update the line endings.